### ISSUE TITLE
offtopic - statsvn

### ISSUE THREAD

**Issue by il--ya** [Appendium/objectlabkit#25]
@benoitx, sorry about offtopic request, but it seems you don't respond on sourceforge, so I'm trying to reach out here. Any chance of you looking at your child statsvn, [issue 109](https://sourceforge.net/p/statsvn/bugs/109/)? If you just remove this broken version check that would suffice...

**Comment by benoitx** [comment]
Hi Illya

I’m afraid that the developers of StatSVN have “moved on”... are you interested in becoming a committer? That might be the easiest way.

I’m not sure that I could even check in a patch from you...

Let me know

Sorry

Benoît 

-----
Important Notice
This communication contains information that is considered
confidential and may also be privileged. It is for the exclusive use of the intended recipient(s). If you are not the intended recipient(s) please note that any form of distribution, copying or use of this communication or the
information in it is strictly prohibited and may be unlawful. If you have received this communication in error please return it to the sender and
delete the original

**Comment by il--ya** [comment]
Hi Benoît,

Thank you for your reply. I struggled with this project over this weekend
(no java experience whatsoever). Spent two days trying first to get it
built under old maven 1.1 or 1.0 by manually downloading dependencies, then
finally give up on that idea and created pom.xml and got it building under
maven 3.6.1. Now I can run it, but it doesn't work properly complaining
that "The repository object is not valid." Original version works fine with
the same log :(
I gave up for now.
If I ever get it working, I will probably just move it to github, as I'm
more used to it than sourceforge.

Cheers,
Ilya
